KomodoPlatform/atomicDEX-Desktop
# show activated if activated

Fixes a bug where you can't enable a coin because when you do the API response in logs is 
```
[20:48:23] [debug] [mm2.service.cpp:1049] [922060]: error: bad answer json for enable/electrum details: {
    "error": "rpc:212] dispatcher_legacy:160] lp_commands_legacy:121] lp_coins:2270] Coin ZRX-PLG20 already initialized"
}
```
In this case, PR will make sure app adds coin to be visible in portfoilio.
To test:
- launch app, and get rpc password from `settings > security > view seed and private key`
- use rpc password to [activate coins in CLI](https://developers.komodoplatform.com/basic-docs/atomicdex-api-legacy/coin_activation.html). To cover all cases, include at least one of each: UTXO / ERC-20 / SLP / ZHTLC
- Coins will not be visible in app. Select them for enabling from coin activation menu.
- Confirm coin is now visible in portfolio, and when you go to wallet page and click "Receive" it shows correct address and not `Invalid Ticker` (you might see this if you are fast enough, but address should appear within 3-5 sec at most)
